docs/tutorials/stateful-application/cassandra.md

```diff
@@ -39,343 +39,147 @@ import it to the website.
-  - metadata:
-      name: cassandra-data
-      annotations:
-        volume.beta.kubernetes.io/storage-class: fast
-    spec:
-      accessModes: [ "ReadWriteOnce" ]
-      resources:
-        requests:
-          storage: 1Gi
----
-kind: StorageClass
-apiVersion: storage.k8s.io/v1beta1
-metadata:
-  name: fast
-provisioner: kubernetes.io/gce-pd
-parameters:
-  type: pd-ssd
+```shell
+kubectl create -f cassandra-statefulset.yaml
 ```
 
-[Download example](https://raw.githubusercontent.com/kubernetes/examples/master/cassandra-statefulset.yaml)
-<!-- END MUNGE: EXAMPLE cassandra-statefulset.yaml -->
+{% include code.html language="yaml" file="cassandra-statefulset.yaml" ghlink="/docs/tutorials/stateful-application/cassandra-statefulset.yaml" %}
 
-Create the Cassandra StatefulSet as follows:
+## Validating The Cassandra StatefulSet
 
-```console
-$ kubectl create -f cassandra/cassandra-statefulset.yaml
-```
-
-## Step 3: Validate and Modify The Cassandra StatefulSet
-
-Deploying this StatefulSet shows off two of the new features that StatefulSets provides.
+1. Get the Cassandra StatefulSet:
 
-1. The pod names are known
-2. The pods deploy in incremental order
-
-First validate that the StatefulSet has deployed, by running `kubectl` command below.
-
-```console
-$ kubectl get statefulset cassandra
+```shell
+kubectl get statefulset cassandra
 ```
 
-The command should respond like:
+   The response should be
 
 ```console
 NAME        DESIRED   CURRENT   AGE
-cassandra   3         3         13s
+cassandra   3         0         13s
 ```
 
-Next watch the Cassandra pods deploy, one after another.  The StatefulSet resource
-deploys pods in a number fashion: 1, 2, 3, etc.  If you execute the following
-command before the pods deploy you are able to see the ordered creation.
+   The StatefulSet resource deploys pods sequentially.  
 
-```console
-$ kubectl get pods -l="app=cassandra"
+{:start="2"}
+2. Get the Pods to see the ordered creation status:
+
+```shell
+kubectl get pods -l="app=cassandra"
 NAME          READY     STATUS              RESTARTS   AGE
 cassandra-0   1/1       Running             0          1m
 cassandra-1   0/1       ContainerCreating   0          8s
 ```
 
-The above example shows two of the three pods in the Cassandra StatefulSet deployed.
-Once all of the pods are deployed the same command will respond with the full
-StatefulSet.
+**Note:** It can take up to ten minutes for all three pods to deploy. 
 
-```console
-$ kubectl get pods -l="app=cassandra"
+Once all pods are deployed, the same command returns:
+
+```shell
+kubectl get pods -l="app=cassandra"
 NAME          READY     STATUS    RESTARTS   AGE
 cassandra-0   1/1       Running   0          10m
 cassandra-1   1/1       Running   0          9m
 cassandra-2   1/1       Running   0          8m
 ```
 
-Running the Cassandra utility `nodetool` will display the status of the ring.
+Running the Cassandra utility `nodetool` displays the status of the ring.
```

Comment:
Rewrite: 3. Run the Cassandra utility `nodetool` to display the status of the ring